Documentation/Settings.cfg

```diff
@@ -3,25 +3,25 @@
 
 [general]
 
-project     = <project>
-version     = <version>
-release     = <release>
-copyright   = <copyright>
+project     = Content and Marketing Guide
+version     = 1.0
+release     = 1.0
+copyright   = TYPO3 Project Contributors
 
 [html_theme_options]
 
 # "Edit on GitHub" button
-github_repository    = <github-repository>
-github_branch        = <github-branch>
+github_repository    = https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-ContentAndMarketing
+github_branch        = main
 
 # Footer links
-project_home         = <project-home>
-project_contact      = <project-contact>
-project_repository   = <project-repository>
-project_issues       = <project-issues>
-project_discussions  = <project-discussions>
+project_home         = https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-ContentAndMarketing
```

Comment:
Missed this one: The project home of a standalone manual should be the docs.typo3.org URL:
```
project_home = https://docs.typo3.org/m/typo3/guide-contentandmarketing/main/en-us/
```
See [here](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#footer-links) for further details.